subcommittees/README.md

```diff
@@ -21,6 +21,7 @@ task).
 * [Finance](finance/README.md)
 * [Lesson Organization and Develpment](lessons/README.md)
 * [Partner Relations and Fundraising](partner-relations/README.md)
+* [Code of Conduct and Policy](policy-CoC/README.md)
```

Comment:
I think we should have a consistent name.  Currently you have “Code of Conduct and Policy” here and “Code of Conduct Policy Subcommittee” in the subcommittee's README.  I'd recommend the name be just “Code of Conduct Subcommittee” with a slug of just `CoC`.  Once you've scoped it to “Code of Conduct” I don't think we need to further scope it to “policy” (especially since the README also tasks the subcommittee with enforcement).  And I don't think you want to scope it to “policy” without restricting it to “Code of Conduct” (there are lots of Carpentry policies which do not fall under this subcommittee).